### PR TITLE
Update /jobs and /jobs/[slug] page to pull from teamtailor api

### LINKED ISSUE #773: Update /jobs and /jobs/[slug] page to pull from teamtailor api
**What's the problem? (required)** 
Currently, the jobs page pull data from the Sanity CMS. We'd like to instead pull from teamtailor using its [API](https://docs.teamtailor.com/). For capturing applications, the email call to action should be replaced with a form and all fields should be posted to teamtailor using its API.

### PR DESCRIPTION
**Pull request by FullStackTiger**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

- Update /jobs and /jobs/[slug] page to pull from teamtailor api

**Does this pull request close any open issues?**

- Fixes #773 

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/www/content/index.tsx [hunk 1]
```diff
@@ -123,3 +123,11 @@ export const DashboardStreamHealth = {
     url: "https://livepeer.com/",
   },
 };
+
+export const Jobs = {
+  metaData: {
```
Comment (adamsoffer) [inline packages/www/content/index.tsx:128]: Update meta data for /jobs page.
Title: "Jobs - Livepeer.com"
Description: "Join Us. From Anywhere."

File: packages/www/pages/jobs.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+type jobState = {
+  id: string;
+  title: string;
+  slug: string;
+};
+
+const JobsPage = () => {
+  const [jobs, setJobs] = useState<jobState[]>([]);
+  const [loading, setLoading] = useState(true);
```
Comment (adamsoffer) [inline packages/www/pages/jobs.tsx]: Can we use SSG for this page?

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: packages/www/pages/jobs/[slug].tsx [hunk 3]
```diff
@@ -140,15 +132,9 @@ const Page = ({
...
-    "https://dp4k3mpw.api.sanity.io/v1/graphql/production/default",
-    print(allJobs),
-    {
-      where: {},
-    }
-  );
+  const allJob = await getJobs();
   let paths = [];
-  allJob.map((page) => paths.push({ params: { slug: page.slug.current } }));
+  allJob.map((page) => paths.push({ params: { slug: page.id } }));
```
Comment (adamsoffer) [inline packages/www/pages/jobs/[slug].tsx]: @FullStackTiger is it possible to use a pretty slug for this? ie `livepeer.com/jobs/protocol-engineer`? If not, that's okay. But would be nice.

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: packages/www/pages/jobs/[slug].tsx (hunk 3 above)
Comment (FullStackTiger) [reply packages/www/pages/jobs/[slug].tsx]: If the title contains special symbols like "Marketing Manager / Sr. Marketing Manager", it is complicated a little.

File: .env.local.example [hunk 4]
```diff
@@ -5,7 +5,7 @@ NEXT_PUBLIC_GA_TRACKING_ID=
 NEXT_PUBLIC_SEGMENT_WRITE_KEY=
 NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=
 NEXT_PUBLIC_STRIPE_ENABLED_IN_DEV_MODE=
-SANITY_API_TOKEN=
+NEXT_PUBLIC_SANITY_API_TOKEN=
```
Comment (adamsoffer) [inline .env.local.example]: Change back to `SANITY_API_TOKEN` and add `TEAMTAILOR_API_TOKEN`

File: packages/www/pages/jobs.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
+import { useRouter } from "next/router";
+import Fade from "react-reveal/Fade";
+import { Element } from "react-scroll";
+import { Box } from "@livepeer.com/design-system";
+import Hero from "components/Marketing/Hero";
+import Layout from "layouts/main";
+import { getJobs } from "hooks";
```
Comment (adamsoffer) [inline packages/www/pages/jobs.tsx]: `getJobs` isn't being used as a hook. Let's define somewhere else. Also, hooks should be prefixed with `use`.

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: packages/www/components/Marketing/JobApplicationForm/index.tsx [hunk 6]
```diff
@@ -0,0 +1,437 @@
...
+                      transition: "border .24s ease-in-out",
+                      minWidth: "296px",
+                      minHeight: "70px",
+                      display: "flex",
+                      justifyContent: "center",
+                      alignItems: "center",
+                    }}>
+                    {loadingPdf
+                      ? "Uploading..."
+                      : "Drag and Drop your CV file or upload files here"}
```
Comment (adamsoffer) [inline packages/www/components/Marketing/JobApplicationForm/index.tsx]: Change text to "Drag and drop your CV file or upload file here" (file should be singular).

**Review by adamsoffer — CHANGES_REQUESTED** [review]

**Pull request merged**